packages/cli/test/cli.test.js

```diff
@@ -98,6 +98,28 @@ describe('Lighthouse CI CLI', () => {
         .replace(log.reset, '');
       projectToken = tokenSentence.match(/Use token ([\w-]+)/)[1];
     }, 30000);
+
+    it('should create a new project with config file', async () => {
+      const wizardProcess = spawn('node', [CLI_PATH, 'wizard', `--config=${rcFile}`]);
+      wizardProcess.stdoutMemory = '';
+      wizardProcess.stderrMemory = '';
+      wizardProcess.stdout.on(
+        'data',
+        chunk => (wizardProcess.stdoutMemory += chunk.toString().replace(/\n/g, ''))
+      );
+      wizardProcess.stderr.on('data', chunk => (wizardProcess.stderrMemory += chunk.toString()));
+
+      await waitForCondition(() => wizardProcess.stdoutMemory.includes('Which wizard'));
+      await writeAllInputs(wizardProcess, [
+        '', // Just ENTER key to select "new-project"
+        '', // Just ENTER key to use serverBaseUrl from config file
+        'AwesomeCIProjectName', // Project name
+        'https://example.com', // External build URL
+      ]);
+
+      expect(wizardProcess.stdoutMemory).toContain('https://github.com/GoogleChrome/lighthouse-ci');
```

Comment:
I don't really understand these expectations, don't we want to test that the correct `serverBaseUrl` was hit?

WDYT about changing the test above to write to a tmp file that uses `http://localhost:${server.port}` as the `serverBaseUrl`?

Reply:
Hm, yeah I'm not sure that first case is worth testing given our current set of options. I'd vote to just modify the existing test to pass in `http://localhost:${server.port}` through a config.